lld/ELF/Arch/AMDGPU.cpp

```diff
@@ -79,8 +79,8 @@ uint32_t AMDGPU::calcEFlagsV4() const {
 
     if (retXnack == EF_AMDGPU_FEATURE_XNACK_UNSUPPORTED_V4 ||
         (retXnack != EF_AMDGPU_FEATURE_XNACK_ANY_V4 &&
-            (getEFlags(f) & EF_AMDGPU_FEATURE_XNACK_V4)
-                != EF_AMDGPU_FEATURE_XNACK_ANY_V4)) {
+         (getEFlags(f) & EF_AMDGPU_FEATURE_XNACK_V4) !=
```

Comment:
drop unneeded reformatting. Use a diff clang-format instead of formatting the whole file
```
 (cd ~/llvm; git diff -U0 'HEAD^' | ~/llvm/clang/tools/clang-format/clang-format-diff.py -p1 -i)
```